models/articles.model.js

```diff
@@ -0,0 +1,15 @@
+const format = require("pg-format");
+const db = require("../db/connection");
+
+exports.selectAllArticles = () => {
+  return db.query(`SELECT * FROM articles`);
+};
+
+exports.selectArticleById = (article_id) => {
+  const articlesByIdQuery = format(
+    `SELECT * FROM articles
+    WHERE article_id = %L`,
+    [article_id]
+  );
+  return db.query(articlesByIdQuery).then(({ rows }) => rows);
```

Comment:
should do a .then block here and then isolate the article obj from the array, and return that object, rather than the array with one article obj inside..